README.md

```diff
@@ -1,6 +1,45 @@
 # webvr-polyfill-dpdb
 
-This is a fork of the [online cache](https://storage.googleapis.com/cardboard-dpdb/dpdb.json) of the Device Parameter Database (DPDB) for the [WebVR Polyfill](https://github.com/googlevr/webvr-polyfill).
+This is a fork of the [online cache](https://storage.googleapis.com/cardboard-dpdb/dpdb.json) of the Device Parameter Database (DPDB) for the [webvr-polyfill].
+
+## Adding A Device
+
+You'll need to update `dpdb-formatted.json` with your device's information in the following format:
+
+```json
+{
+  "type":"android",
```

Comment:
can we run this through `JSON.stringify(JSON.parse(…))` to properly format spaces?